dafni_cli/commands/delete.py

```diff
@@ -144,6 +184,57 @@ def dataset(ctx: Context, version_id: List[str]):
     click.echo("Datasets deleted")
 
 
+def collate_dataset_version_details(
+    session: DAFNISession, version_id_list: List[str]
+) -> Tuple[List[str], List[str]]:
+    """
+    Checks for destroy privileges for the user, and produces a list of the
+    details of each dataset to be deleted
+
+    Args:
+        session (DAFNISession): User session
+        version_id_list (List[str]): List of the dataset version IDs of each
+                                     dataset to be deleted
+
+    Returns:
+        List[str]: List of the dataset details to be displayed during deletion
+                   confirmation
+        List[str]: Dataset IDs that should be deleted
+    """
+    return _collate_dataset_details(
+        session=session,
+        version_id_list=version_id_list,
+        obtain_details=lambda dataset_meta: dataset_meta.get_dataset_version_details(),
+        permissions_message="You do not have sufficient permissions to delete dataset version:",
+    )
+
+
+@delete.command(help="Delete one or more dataset versions")
+@click.argument("version-id", nargs=-1, required=True, type=str)
+@click.pass_context
+def dataset_version(ctx: Context, version_id: List[str]):
```

Comment:
I agree its confusing to a developer but it may be worse for a user if I change it as click uses the name in the help text e.g:
![image](https://github.com/dafnifacility/cli/assets/90245114/965aab84-4f2f-489c-8d61-3b66cfca7938)

Reply:
Actually thinking about it version-ids would be better